fastlane/Fastfile

```diff
@@ -17,4 +17,15 @@ platform :ios do
       test | xcpretty
       ")
   end
+
+  desc "Runs SwiftLint and returns any errors if our linter rule aren't met"
+  lane :lint do
+    swiftlint(
+      mode: :lint,
+      config_file: ".swiftlint.yml",
+      raise_if_swiftlint_error: true,
+      ignore_exit_status: true,
```

Comment:
🤔  Don't we want this to be false? We don't want to ignore the exist status code for this job?

Reply:
Ope, yep you're correct, [this defaults to false](https://docs.fastlane.tools/actions/swiftlint/), so I'll remove this line